src/Common.Lib/XmlDocWrapper.cpp

```diff
@@ -0,0 +1,194 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "stdafx.h"
+#include "XmlDocWrapper.h"
+#include "systemstring.h"
+
+#ifndef PLATFORM_UNIX
+using namespace ATL;
+#endif
+
+namespace CommonLib
+{
+    CXmlDocWrapper::CXmlDocWrapper()
+    #ifdef PLATFORM_UNIX
+        : m_cleanupDoc(),
+        m_pDocument(nullptr)
```

Comment:
>m_pDocument(nullptr) [](http://example.com/codeflow?start=8&length=20)

should `m_pDocument` be initialized for windows also?

Reply:
Doesn't need to be because it isn't a pointer. CComPtr will initualize it to null. A cleaner way to do this might be to use initialization in the header file, though.